contributing/index.md

```diff
@@ -70,18 +70,22 @@ Now that you have your account set up, you can do the actual upload:
   3. If you're not signed in, click the Sign In button in the top right and sign
   in with your email.
 
-  4. Click Start Review and add a reviewer. See
+  4. Fill out the bug entry properly. Please specify the issue tracker prefix and
+  the issue number, separated by a colon, e.g. webrtc:123 or chromium:12345. If
+  you do not have an issue tracker prefix and an issue number just add `None`.
```

Comment:
We can also move this in step number 1, right after "This will open a text editor...". If I remember correctly the presubmit will stop at that stage asking for a meaningful bug.

Reply:
Yep, much better.